engine/extended_provider/extended_providers.go

```diff
@@ -0,0 +1,164 @@
+package extended_provider
```

Comment:
By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. Please see:
- https://go.dev/blog/package-names
- https://go.dev/doc/effective_go#package-names

Reply:
Fixed